### PR TITLE
Add functionality for additional SOCD modes

### PR DESCRIPTION
**Pull request by ScrubTier**
Updated platformio.ini to point to OpenStickFoundation MPG repo.
Added SOCD modes for OLED display.
Added SOCD modes to web configurator pages and recompiled fsdata.c

Have already submitted relevant changes to the OpenStick MPG, these GP2040 changes will fail to compile without MPG being first updated.

### CODE REVIEW

**Comment by TheTrainGoes** [comment]
Hi @ScrubTier ,

We will be moving to 0.4.4 soon and will look into merging these changes.  

Thank you!

**Comment by TheTrainGoes** [comment]
Hi @ScrubTier ,

We jumped to v0.5.0 but it is out in the release section now.  Please let me know what we can do to help get these changes included in v0.5.1

File: src/display.cpp [hunk 1]
```diff
@@ -105,16 +105,41 @@ void setStatusBar(Gamepad *gamepad)
...
+		case DPAD_MODE_LEFT_ANALOG:  statusBar += "      LEFT"; break;
+		case DPAD_MODE_RIGHT_ANALOG: statusBar += "     RIGHT"; break;
 	}
 
 	switch (gamepad->options.socdMode)
 	{
 		case SOCD_MODE_NEUTRAL:               statusBar += "-N"; break;
 		case SOCD_MODE_UP_PRIORITY:           statusBar += "-U"; break;
 		case SOCD_MODE_SECOND_INPUT_PRIORITY: statusBar += "-L"; break;
+		case SOCD_MODE_X_NEUTRAL_Y_NEUTRAL:   statusBar += "-XNYN"; break;
```
Comment (TheTrainGoes) [inline src/display.cpp:118]: The current screen layout only leaves two characters for the SOCD mode.  Do we want to keep descriptions or could we replace them with `01`-`24` and leave the main three as `-N`, `-U`, `-L`?

**Comment by arntsonl** [comment]
Additional SOCD modes will need to be added on our new code base, and include all of the SOCD modes we've discussed on the discord.

**Pull request closed without merging**